demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -1501,20 +1515,13 @@ def _execute_mockable_test(self, proxy: MITMProxy):
                 return False
 
         # Running playback after successful record to verify the record is valid for future runs
-        if status == PB_Status.COMPLETED:
+        if succeed:
```

Comment:
`succeed` might be a bit misleading, i would change to `completed`
also, this variable might potentially be referenced before assignment, so lets init it to be on the safe side

Reply:
Changed to `completed`
But I don't see how it can be referenced before assignment, the indentation is misleading but in case `lock` is `False` I exit the frame.